client/wfprev-war/src/main/angular/src/app/components/search-filter/search-filter.component.ts

```diff
@@ -162,7 +161,56 @@ ngOnInit(): void {
     }
     this.emitFilters();
   }
-  
+
+  syncAllWithItemToggle(
+    event: MatOptionSelectionChange,
+    value: any,
```

Comment:
Can it be a defined type vs any? We want to avoid the use of any if at all possible.

Reply:
Same comment for the `any` vars below as well. We should never need to use any, and it's generally not an ideal practice as it leads to bugs